pyro/distributions/torch/bernoulli.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, division, print_function
+
+import torch
+import torch.nn.functional as F
+
+from pyro.distributions.bernoulli import Bernoulli as _Bernoulli
+from pyro.distributions.torch_wrapper import TorchDistribution
+from pyro.distributions.util import copy_docs_from, get_clamping_buffer
+
+
+@copy_docs_from(_Bernoulli)
+class Bernoulli(TorchDistribution):
+    enumerable = True
+
+    def __init__(self, ps=None, logits=None, *args, **kwargs):
+        if (ps is None) == (logits is None):
+            raise ValueError("Got ps={}, logits={}. Either `ps` or `logits` must be specified, "
+                             "but not both.".format(ps, logits))
+        if ps is None:
+            ps = F.sigmoid(logits)
+        eps = get_clamping_buffer(ps)
+        ps = ps.clamp(min=eps, max=1-eps)
+        torch_dist = torch.distributions.Bernoulli(ps)
```

Comment:
Do any tests fail if we do not do this clamping here?

Reply:
Yes, your `_flow` tests fail. I'm glad you implemented them.

Reply:
Ahh..I see. Makes sense! Will add logit support in PyTorch soon.